src/main/java/com/mongodb/jdbc/MongoSQLDatabaseMetaData.java

```diff
@@ -39,7 +39,10 @@ public String getTimeDateFunctions() throws SQLException {
     @Override
     public ResultSet getProcedures(
             String catalog, String schemaPattern, String procedureNamePattern) throws SQLException {
-        throw new SQLFeatureNotSupportedException("TODO");
+        // TODO: create result set metadata that describes the columns of this result set
+        //  even though it is empty. This applies to all updated methods in this PR and is
+        //  blocked on SQL-513 and SQL-535.
```

Comment:
Will this be implemented in this PR after SQL-513 and SQL-535 are merged?

Reply:
I plan to do that, yes. If you look at Patrick's 553 PR (#54), that's what I'm going for. At minimum, after 513 (#47) is merged, I'll be able to make the metadata docs and maybe that will be enough for now if 535 (#53) is still in pogress. Perhaps as part of 535 you could change the `return new MongoSQLResultSet(...)` lines in the DatabaseMetaData methods to whatever new constructor signature you make in that ticket. We'll have to see how everything plays out.